### PR TITLE
Fault injection in Tensor operators

### PR DESCRIPTION
**Pull request by uditagarwal97**

### CODE REVIEW

File: llvm_passes/CustomTensorOperatorInstSelector.cpp [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+#include <unordered_map>
+#include <cassert>
+
+using namespace llvm;
+
+namespace llfi {
+
+
+static cl::list< std::string > layerNo("layerNo",
+         cl::desc("Layer Number in which you want to inject bitflip faults. Pass 0 for injecting faults in all the layers.\n Semi-colon seperated values. Example: 1;0;2"),
```
Comment (abrahamchan) [inline llvm_passes/CustomTensorOperatorInstSelector.cpp:23]: For LLVM source code, it would be preferable to keep all the lines within 80 columns.

**Review by abrahamchan — COMMENTED** [review]
@uditagarwal97 I have conducted a code review and left a few comments for you. Thanks!

File: tools/json-c-setup.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```
[the remark this answers is not in the document]
Comment (uditagarwal97) [reply tools/json-c-setup.sh:1]: Json-c will be used along with all the benchmarks, that is why I kept it inside the tools folder.

**Review by abrahamchan — APPROVED** [review]
LGTM

**Pull request merged**